source/slang/hlsl.meta.slang

```diff
@@ -19803,7 +19803,8 @@ ${
             return spirv_asm {
                 OpCapability ImageFootprintNV;
                 OpExtension "SPV_NV_shader_image_footprint";
-                %resultVal:$$__FootprintData = OpImageSampleFootprintNV &this $coords $granularity $useCoarseLevel;
+                %sampledImage : __sampledImageType(this) = OpSampledImage $this $sampler;
+                %resultVal:$$__FootprintData = OpImageSampleFootprintNV %sampledImage $coords $granularity $useCoarseLevel;
```

Comment:
Don't we need to make the same changes to the other functions?
As an example, the line 19834 has the same pattern as this line.

Reply:
thanks, good spot